libdctlint/record.go

```diff
@@ -75,6 +75,10 @@ func (conf *Conf) checkRecord(
 			rlog.Warn().Msg("record txtConflictMatchingPrefix is not defined")
 			exitVal |= exitvals.CheckWarn
 		}
+		if strings.HasPrefix(record.Data, "v=spf1") {
+			rlog.Info().Msg("use SPFM instread of TXT for spf records")
```

Comment:
Typo. I'd suggest also to add reference to the specification.
My proposal:
"It is recommended to use SPFM instead of bare SPF record to allow for merging several services. See specification section 6.10 for details."

Reply:
> See specification section 6.10 for details

That makes me think this linter could and should imitate how ShellCheck is using SC codes, such as  SC1118 in example below.

https://github.com/koalaman/shellcheck/wiki

With that log lines could look something like:

      rlog.Info().Str("moreinfo", "DCTL1042").Msg("recommendation, use SPFM instead of bare SPF record")

Where long explanations can be found from https://github.com/Domain-Connect/dc-template-linter/wiki/DCTL1042 Point of these wiki pages should be that they tell very verbosely what is wrong, propose common fixes to issues, and have reference links to specification, RFCs, and other reasoning why the advise is what it is. See links below for a pretty good example of not shying away from being verbose.

https://github.com/koalaman/shellcheck/wiki/SC2039
https://github.com/koalaman/shellcheck/wiki/SC2044

Assuming this proposal sounds reasonable there might be need for fine grain spec subtitling changes, but that's just some work that needs doing.

Reply:
So your point is that my suggested text is too verbose and you propose to keep the messages short and add external wiki resources that describe the problem in more detail and offer potential solution.
Wiki approach can be useful as it may be maintained outside of the tool release cycle. I like, but yes it is a bit of work to have an initial spin ready.

Reply:
Sounds like a rough consensus. Let me fix typo in this message, merge the pull, and start DCTL work.

Reply:
First version of DCTL messages is done: https://github.com/Domain-Connect/dc-template-linter/pull/10